astropy/coordinates/tests/test_transformations.py

```diff
@@ -430,3 +430,9 @@ def test_function_transform_with_differentials():
         t2 = t3.transform_to(TCoo2)
         assert len(w) == 1
         assert 'they have been dropped' in str(w[0].message)
+
+
+def test_lookup_name():
+    g = t.TransformGraph()
+    with pytest.raises(NameError):
+            g.lookup_name('banana', raise_exception= True)
```

Comment:
Could you use 4 spaces as indentation here? And for keyword-arguments there's generally no whitespace around the`=`.